datastore/sql/sqlitedsdb.nim

```diff
@@ -279,25 +279,51 @@ proc getDBFilePath*(path: string): ?!string =
   except CatchableError as exc:
     return failure(exc.msg)
 
-proc close*(self: SQLiteDsDb) =
-  self.containsStmt.dispose
-  self.getStmt.dispose
-  self.beginStmt.dispose
-  self.endStmt.dispose
-  self.rollbackStmt.dispose
-  self.getVersionedStmt.dispose
-  self.updateVersionedStmt.dispose
-  self.insertVersionedStmt.dispose
-  self.deleteVersionedStmt.dispose
-  self.getChangesStmt.dispose
+proc close*(self: var SQLiteDsDb) =
```

Comment:
It doesn't look like `self` needs to be a var, but it does seem to indicate intent a bit better.

Reply:
Ah nevermind, I see the nil setter down below.